src/Web/Views/Shared/_Layout.cshtml

```diff
@@ -60,19 +62,26 @@
     <header class="navbar navbar-light navbar-static-top">
         <div class="container">
             <article class="row">
-                <section class="col-lg-6 col-md-5 col-xs-11">
+                <section class="col-lg-2 col-md-4 col-xs-11">
                     <a asp-area="" asp-controller="Catalog" asp-action="Index" class="navbar-brand">
                         <img src="~/images/brand.png" alt="Microsoft Fraud Protection Merchant Application" title="@apiBaseUrl" />
                     </a>
                 </section>
                 <partial name="_LoginPartial" />
+                @using (Html.BeginForm("Update", "Environment", FormMethod.Post, new { id = "environmentupdate" }))    
```

Comment:
Anything we can do to make the header look a bit nicer?  e.g.

- e.g. use flex to align things better?
- make the cart icon be centered between the menu options and the env input?
- get the menu option text and env input vertically aligned?
- maybe make the env id form smaller or something?

Reply:
fixed issues related to overlapping, it looks better than before but let me know if you still see issues.

Reply:
Sounds good.  Thanks.